cl/users/templates/profile/settings.html

```diff
@@ -150,6 +150,18 @@
                     </p>
                 {% endif %}
             </div>
+            <div class="form-group">
```

Comment:
This is fine and works for now, but I'd like to refactor the alerts tab in people's profiles and move this toggle there. It makes sense as part of the UserProfile model, but it doesn't make as much sense in the user profile webpage.

Reply:
Sure, I'm working on this change as part of the refactor tabs PR.